src/components/Balances.jsx

```diff
@@ -93,11 +93,13 @@ const Balances = ({ entity }) => {
             </tbody>
           </table>
           <Row className="p-2 mb-4" justify="space-between" style={{ fontSize: '0.8rem' }}>
-            <Col>
```

Comment:
Please add a boolean attribute named `showGasUsed` and pass to component. It' not nice to be dependent on datatype in component